bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -202,6 +205,9 @@ spec:
         displayName: Conditions
         path: conditions
       version: v1alpha1
+    - kind: KfDef
```

Comment:
need remove

Reply:
It gets auto generated when I do `make bundle`